tfhe/src/core_crypto/entities/lwe_compact_public_key.rs

```diff
@@ -183,3 +184,16 @@ impl<Scalar: UnsignedInteger> LweCompactPublicKeyOwned<Scalar> {
         Self::from_container(vec![fill_with; 2 * lwe_dimension.0], ciphertext_modulus)
     }
 }
+
+impl<C: Container> ParameterSetConformant for LweCompactPublicKey<C>
+where
+    C::Element: UnsignedInteger,
+{
+    type ParameterSet = GlweCiphertextConformanceParameters<C::Element>;
+
+    fn is_conformant(&self, parameter_set: &Self::ParameterSet) -> bool {
+        let Self { glwe_ciphertext } = self;
+
+        glwe_ciphertext.is_conformant(parameter_set)
```

Comment:
check power of 2 here for the polynomial size (equivalent to the lwe dim) and that the GlweDim = 1